roles/node_images_ncn/files/cloud/templates/cloud-init-network.mlx.tmpl

```diff
@@ -0,0 +1,31 @@
+## template:jinja
+network:
+  version: 2
+  ethernets:
+    mgmt0:
+      dhcp6: no
+      mtu: 9000
+    mgmt1:
+      dhcp6: no
+      mtu: 9000
+  bonds:
+    bond0:
+      interfaces: [mgmt0, mgmt1]
+      dhcp4: false
+      mtu: 9000
+      parameters:
+        mode: 802.3ad
+        mii-monitor-interval: 100
+        lacp-rate: fast
+        ad-select: bandwidth
+        transmit-hash-policy: encap3+4
```

Comment:
If this is just for a workaround that's fine.  Encap 3+4 is not LACP compliant so this could be a risk and cause other issues.

Reply:
Doesn't the current WAR apply this for Mellanox?

Reply:
This is certainly not the solution for CASMTRIAGE-5839.   But given that layer2 doesn't work at all with Mellanox NICs, isn't this at least useful for a WAR until we get a fix?   I don't think we should ship 1.5 with this.

Reply:
I don't know how bad the issues are that we expect to hit with encap3+4 to trade-off against the issue we hit using layer2+3 or layer2 on Mellanox systems.    I know that we consistently have storage boot issues when it is set to layer2+3 or layer2.   Therefore, every time we do an 1.5 install or upgrade on fanta (or drax) we need to stop to apply the WAR.

Reply:
Therefore, we have to use encap3+4 every time we install or upgrade 1.5 anyway.